app/views/users/_form.html.haml

```diff
@@ -20,25 +20,36 @@
           = f.phone_field :phone,
            placeholder: '+14135450056', class: 'form-control'
       .col-lg-6
+        = field_set_tag nil, class: 'form-group' do
+          %legend.m-0 Rosters
+          = f.hidden_field :'roster_ids[]', value: '', id: nil
+          .list-group.list-group-form
+            - user_roster_options = @user.rosters | @current_user.memberships.where(admin: true).map(&:roster)
+            = f.collection_check_boxes :roster_ids, user_roster_options, :id, :name, include_hidden: false do |c|
+              - user_roster_permitted = @current_user.admin_in? c.object
+              = c.label class: ['m-0', 'list-group-item', user_roster_permitted ? 'list-group-item-action' : nil] do
+                .custom-checkbox.custom-control
+                  = c.check_box checked: @user.membership_in(c.object).present?,
+                                disabled: !user_roster_permitted,
+                                class: 'custom-control-input'
+                  = c.label class: 'custom-control-label'
```

Comment:
HTML spec technically says that `<label>`s shouldn't have `<label>`s inside of them.

Reply:
That may have to be a larger conversation, this was at @benmelz suggestion, following the use of this format multiple times elsewhere

Reply:
Yeah, I mean, that's why I said "technically". But:

> **Permitted content**: Phrasing content, but no descendant `label` elements. No labelable elements other than the labeled control are allowed. 

Surrounding a checkbox with a label is fine, but you shouldn't need a second label inside. This seems to work:

```suggestion
                  .custom-control-label= c.object.name
```

Reply:
Slightly better

```suggestion
                  .custom-control-label= c.text
```